javascript/builtins/AsyncIterator.json

```diff
@@ -0,0 +1,952 @@
+{
+  "javascript": {
+    "builtins": {
+      "AsyncIterator": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/AsyncIterator",
+          "support": {
+            "chrome": {
+              "version_added": "63"
+            },
+            "chrome_android": {
+              "version_added": "63"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "57"
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "nodejs": {
+              "version_added": "10.0.0"
+            },
+            "opera": {
+              "version_added": "50"
+            },
+            "opera_android": {
+              "version_added": "46"
+            },
+            "safari": {
+              "version_added": "11.1"
+            },
+            "safari_ios": {
+              "version_added": "11.3"
+            },
+            "samsunginternet_android": {
+              "version_added": "8.0"
+            },
+            "webview_android": {
+              "version_added": "63"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "AsyncIterator": {
+          "__compat": {
+            "description": "<code>AsyncIterator()</code> constructor",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/AsyncIterator/AsyncIterator",
+            "support": {
+              "chrome": {
+                "version_added": false
```

Comment:
@mathiasbynens, does this really not have a constructor in Chrome?

Reply:
And please glance through the other members.

Reply:
> @mathiasbynens, does this really not have a constructor in Chrome?

Not yet. @ExE-Boss's top post explains why.